### DISCUSSION TITLE
USWDS - Feature: Support responsive breakpoint prefix for text alignment

### DISCUSSION THREAD

**Discussion by fallenturtle** [uswds/uswds#6935] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I want to use different text alignment depending on the viewport width.

### Describe the solution you'd like

Support responsive breakpoint prefix for text alignment

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by amyleadem** [comment, marked as answer]
Hi @fallenturtle, 
Are you using USWDS 3? If so, this functionality should already exist in USWDS. You should be able to add the following code to your [USWDS settings configuration](https://designsystem.digital.gov/documentation/settings/#configuring-custom-uswds-settings) in Sass to turn on responsive breakpoints for the [text alignment utility classes](https://designsystem.digital.gov/utilities/paragraph-styles/#text-align):

```
@use "uswds-core" with (
  $text-align-settings: (
    responsive: true
  )
);
```

You can learn more about using responsive variants on our [paragraph style utility classes page](https://designsystem.digital.gov/utilities/paragraph-styles/#responsive-variants-2).

 Hopefully that gets you what you are looking for. Please let us know if that gives you any trouble.

**Comment by brunerae** [comment]
Sending this to FFR for potential closure.

**Comment by ryparker** [comment]
Responsive text alignment is already supported, as the answer above explains. I've moved this to Q&A and marked that explanation as the answer so others can find it. Follow-up questions are welcome here.
